source/_integrations/saj.markdown

```diff
@@ -19,7 +21,15 @@ The WiFi module requires a username and password for authentication where the et
 
 ## Configuration
 
-To enable this sensor, add the following lines to your `configuration.yaml` file:
+1. In the **Configuration** > **Integrations** menu, click the `+ ADD INTEGRATION` button on the lower-right of the screen.
+2. Select **SAJ Solar Inverter** from the pop-up menu.
+3. Complete the dialog box and click `SUBMIT`. 
+
+Note: You should set up fixed/static IP address for the inverter.
+
+## Manual configuration
+
+If you prefer to set up the integration in `configuration.yaml`, add the following lines:
```

Comment:
The manual configuration method should be deprecated and removed from documentation.

Reply:
So... Should I mark it as deprecated or should I remove it?

Reply:
The code should throw deprecation warnings, in the documentation, it should be removed.

Reply:
Thanks!